velox/exec/Driver.cpp

```diff
@@ -1090,6 +1023,68 @@ folly::dynamic Driver::toJson() const {
   return obj;
 }
 
+template <typename Func>
+auto Driver::withDeltaCpuWallTimer(
+    Operator* op,
+    TimingMemberPtr timingMember,
+    Func&& timedFunction) -> decltype(timedFunction()) {
+  // If 'trackOperatorCpuUsage_' is true, create and initialize the timer object
+  // to track cpu and wall time of the timedFunction.
+  auto timer = trackOperatorCpuUsage_
+      // The delta CpuWallTiming object would be recorded to the corresponding
+      // timingMember upon destruction of the timer when withDeltaCpuWallTimer
+      // ends.
+      ? std::make_unique<
```

Comment:
We don't want the unnecessary heap allocation with `unique_ptr`.  The timer can be allocated on stack directly.